pkg/crypto/crypto.go

```diff
@@ -284,6 +285,19 @@ func DefaultCiphers() []uint16 {
 	}
 }
 
+// WhitelistEtcdCipherSuites whitelists valid ciphers for use with etcd.
+func WhitelistEtcdCipherSuites(cipherSuites []string) []string {
```

Comment:
why is this function in library-go?

Reply:
Thought was that apiserver should also define whitelisted ciphers in clientv3 tlsConfig so having this here would allow reasonable reuse but I can put it upstream etcd.